README.md

```diff
@@ -51,6 +51,18 @@ Community (college) maintained list of Algorithms and Data Structures implementa
 | [Stack](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) | | | [\[X\]](stack/Stack.java) | [\[X\]](stack/stack.py) | [\[X\]](stack/stack.go) | [\[X\]](stack/stack.js) |
 
 
+## How to run them
+| Language | Steps |
+|:--------------|:----------------:|
+| C | `gcc -o [filename.c]` |
```

Comment:
Add full steps .. like in C and CPP, add the step to run the output program.
Example

```sh
gcc [filename.c]
./a.out  # unix
./a.exe  # windows
```